bundle/artifacts/wheel/upload.go

```diff
@@ -0,0 +1,68 @@
+package wheel
+
+import (
+	"context"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/databricks/databricks-sdk-go/apierr"
+	"github.com/databricks/databricks-sdk-go/service/workspace"
+)
+
+type upload struct {
+	name string
+}
+
+func Upload(name string) bundle.Mutator {
```

Comment:
Would uploading of wheels also work when a wheel file is referenced from an asset? Like from the list of libraries in a cluster or job?